ui/src/app.jsx

```diff
@@ -33,30 +34,35 @@ export default React.createClass({
 
   routes: {
     '/': 'home',
+    '/demos': 'demos',
 
-    // Variations on learning experiences
-    '/message_popup*': 'messagePopupRedirect',
-    '/teachermoments': 'messagePopup',
-    '/playtest/:cohortKey': 'messagePopupPlaytest',
+    // Included in /demos, shared externally, or used in playtests
+    '/teachermoments/original': 'messagePopup',
     '/teachermoments/bias': 'biasHome',
     '/teachermoments/alpha': 'alphaPlaytest',
-    '/teachermoments/turk-0000': 'turk0000',
     '/teachermoments/mentoring': 'mentoringPlaytest',
     '/teachermoments/mindset': 'mindsetPlaytest',
-    '/teachermoments/danson': 'dansonPlaytest',
     '/teachermoments/danson2': 'dansonPlaytest2',
     '/teachermoments/twine': 'messagePopupTwine',
     '/teachermoments/demo': 'messagePopupDemo',
     '/teachermoments/sub': 'messagePopupPairs',
     '/teachermoments/darius': 'messagePopupDarius',
+    '/teachermoments/chat': 'chatPrototype',
+
+    // Deprecated experiences
+    '/playtest/:cohortKey': 'messagePopupPlaytest',
```

Comment:
@kesiena115 I believe we'll want to deprecate and remove these two experiences, is that right?  This PR doesn't change anything about them right now, just confirming my understanding.